docs/triggers_rules.md

```diff
@@ -141,3 +142,6 @@ As an example, create a rule that calls the hello action whenever a location upd
   You see that the hello action received the event payload and returned the expected string.
 
   You can create multiple rules that associate the same trigger with different actions.
+
+The trigger and action that make a rule must be in the same namespace, which is in fact private.
```

Comment:
I'm going to change to " The trigger and action that make a rule must be in the same namespace."

>and must belong to the default package
What does this means?
You want me to add it, I don't know what a default package is.